platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteIndicator.java

```diff
@@ -0,0 +1,28 @@
+package com.softicar.platform.dom.elements.input.auto;
+
+import com.softicar.platform.dom.elements.DomImage;
+import com.softicar.platform.dom.style.CssPixel;
+import com.softicar.platform.dom.style.CssStyle;
+import com.softicar.platform.dom.styles.CssCursor;
+import com.softicar.platform.dom.styles.CssPosition;
+
+public class DomAutoCompleteIndicator extends DomImage {
+
+	public DomAutoCompleteIndicator(DomAutoCompleteIndicatorType type) {
```

Comment:
the style definitions in here should be extracted to a CSS class

Reply:
extracted